Sources/WebDriver/Requests.swift

```diff
@@ -531,4 +531,18 @@ public enum Requests {
 
         public typealias Response = WebDriverStatus
     }
+
+    public struct SessionWindowHandle: Request {
+        public var session: String 
+        public var many: Bool
+
+        public var pathComponents: [String] { ["session", session, many ? "window_handles" : "window_handle"] }
+        public var method: HTTPMethod { .get }
+
+        public typealias Response = ResponseWithValue<ResponseValue>
+        public struct ResponseValue: Codable {
+            public var windowHandle: String
```

Comment:
Please define two request structs since no response value will have both `windowHandle` and `windowHandles` set at once.